addons/project/views/project_project_views.xml

```diff
@@ -152,12 +152,12 @@
                                         </setting>
                                     </div>
                                 </group>
-                            </group>
-                        </page>
-                        <page name="analytic" string="Analytic" groups="analytic.group_analytic_accounting">
-                            <group>
-                                 <group>
-                                    <field name="account_id"/>
+                                <group name="group_analytic" string="Analytic" col="1" class="row mt16 o_settings_container" groups="analytic.group_analytic_accounting">
+                                    <div class="o_setting_box">
+                                        <group>
+                                            <field name="account_id" class="oe_inline"/>
+                                        </group>
+                                    </div>
```

Comment:
this section should be above the "Tasks Management" as shown in the screenshot and not under "Documents & Analytics"